database/index.js

```diff
@@ -5,6 +5,28 @@ const connection = mysql.createConnection({
   database: 'airbnb_similar_listings',
 });
 
-connection.connect();
+connection.connect((err) => {
+  if (err) {
+    console.log(`Failed to connect to the db, here is the error: ${err}`);
+    return;
+  }
+  console.log(`Successfuly connected to the db, here is the connection id: ${connection.threadId}`);
+});
+
+function getSimilarListings(id, callback) {
+  connection.query(`SELECT similar_listing_id FROM similar_listings WHERE listing_id=${id}`, (err, results) => {
+    if (err) {
+      console.log(`DB query failed, here is the error: ${err}`);
+    }
+
+    const similarListingIds = [];
+    results.forEach((similarListing) => {
+      similarListingIds.push(similarListing.similar_listing_id);
```

Comment:
A suggestion: Do you think using an inner join statement would produce similar results instead creating a new array with all the listing ids?

Reply:
Hmm, I'm not sure, but I will investigate! Thanks for the suggestion.